boms/preview-ee/pom.xml

```diff
@@ -37,7 +37,7 @@
         -->
         <version.jakarta.authorization.jakarta-authorization-api>3.0.0</version.jakarta.authorization.jakarta-authorization-api>
         <version.jakarta.el>6.0.0</version.jakarta.el>
-        <version.jakarta.enterprise.concurrent.jakarta-enterprise.concurrent-api>3.1.0</version.jakarta.enterprise.concurrent.jakarta-enterprise.concurrent-api>
+        <version.jakarta.enterprise.concurrent.jakarta-enterprise.concurrent-api>3.0.4</version.jakarta.enterprise.concurrent.jakarta-enterprise.concurrent-api>
```

Comment:
This is a downgrade, so we shouldn't use this PR as the basis for upgrading standard WildFly.